tools/rust_api/Cargo.toml

```diff
@@ -2,47 +2,47 @@
 name = "kuzu"
 version = "0.9.0"
 description = "An in-process property graph database management system built for query speed and scalability"
-# Note: 1.72 required for testing due to latest dependencies of the arrow feature
-rust-version = "1.67"
+keywords = ["database", "graph", "ffi"]
 readme = "kuzu-src/README.md"
-homepage = "http://kuzudb.com/"
+homepage = "https://kuzudb.com/"
 repository = "https://github.com/kuzudb/kuzu"
 license = "MIT"
 categories = ["database"]
+
+rust-version = "1.81"
+edition = "2021"
+
 # Only include files required to build to keep crate size small
 include = [
     "build.rs",
     "/src",
     "/include",
-# If more files are added to kuzu-src, they should also be added to the rerun-if-changed
-# instructions in build.rs so that rebuilds are detected properly
+    # If more files are added to kuzu-src, they should also be added to the rerun-if-changed
+    # instructions in build.rs so that rebuilds are detected properly
     "/kuzu-src/src",
     "/kuzu-src/cmake",
     "/kuzu-src/third_party",
     "/kuzu-src/CMakeLists.txt",
     "/kuzu-src/tools/CMakeLists.txt",
 ]
 
-edition = "2018"
-
 [dependencies]
-cxx = "1.0"
+arrow = { version = "55", optional = true, default-features = false, features = ["ffi"] }
+cxx = "=1.0.138" # latest version that builds on clang 15
```

Comment:
I'm concerned that we will probably end up unpinning this relatively soon to handle bugs which newer versions of cxx fix, but I guess we can try it and see when stuff breaks.

Reply:
Yeah this is to allow the builds to succeed on our mac runners. Eventually we'll have to either move everything to macOS 15 or figure out a way to update clang on the older version (probably through `brew`?).